### PR TITLE
Update GEP template and process

### PR DESCRIPTION
**Pull request by timebertt**
**How to categorize this PR?**

/area open-source dev-productivity
/kind enhancement
/priority normal

**What this PR does / why we need it**:

This PR updates the GEP template and process description according to the outcome/feedback/experiences with larger contributions over the last few months.

Main changes are:
- added link to the general contribution guidelines to the GEP process description
- added general GEP metadata in the template
- added metadata field and description for "main reviewers"

**Special notes for your reviewer**:

/invite @gardener/gardener-maintainers @stoyanr 

**Release note**:

```other developer
The GEP [template](https://github.com/gardener/gardener/blob/master/docs/proposals/00-template.md) and [process description](https://github.com/gardener/gardener/blob/master/docs/proposals/README.md) was updated. Please take a few minutes to familiarize yourself with the latest changes before working on a GEP.
```

### CODE REVIEW

File: docs/proposals/README.md [hunk 1]
```diff
@@ -27,12 +30,34 @@ GEPs should be created as Markdown `.md` files and are submitted through a GitHu
...
+Apart from general metadata the GEP should name at least one "main reviewer".
+The main reviewers are charged with the following tasks:
+
+- familiarizing themselves with the details of the proposal
+- reviewing the GEP PR itself and any further updates to the document
+- discussing design details and clarifying implementation questions with the author before and after
+ the proposal was accepted
+- reviewing PRs related to the GEP in-depth
+
+Other community members are of course also welcome the help the GEP author, review his work and raise
```
Comment (rfranzke) [inline docs/proposals/README.md]:
```suggestion
Other community members are of course also welcome to help the GEP author, review his work and raise
```

**Review by rfranzke — COMMENTED** [review]
/invite @stoyanr

File: docs/proposals/README.md [hunk 2]
```diff
@@ -27,12 +30,34 @@ GEPs should be created as Markdown `.md` files and are submitted through a GitHu
 
 If not already done, please present your GEP in the [regular community meeting](https://github.com/gardener/documentation/blob/master/CONTRIBUTING.md#weekly-meeting) to brief the community about your proposal (we strive for personal communication :) ). Also consider that this may be an important step to raise awareness and understanding for everyone involved.
 
+The GEP template contains a small set of metadata, which is helpful for keeping track of the enhancement
+in general and especially of who is responsible for implementing and reviewing PRs that are part of
+the enhancement.
+
+### Main Reviewers
+
+Apart from general metadata the GEP should name at least one "main reviewer".
```
Comment (timuthy) [inline docs/proposals/README.md]: Can you elaborate in the text how community members are supposed to find a main reviewer?

**Review by timuthy — CHANGES_REQUESTED** [review]
Thanks for the update.

File: docs/proposals/README.md (hunk 2 above)
Comment (timebertt) [reply docs/proposals/README.md]: Makes sense. Added a sentence, WDYT?

**Review by timuthy — APPROVED** [review]
/lgtm

File: docs/proposals/README.md (hunk 2 above)
Comment (timuthy) [reply docs/proposals/README.md]: Nice, thank you :)

**Review by rfranzke — APPROVED** [review]

**Pull request merged**